grpc_server.go

```diff
@@ -31,14 +31,14 @@ type exportedHardware struct {
 	BondingMode                        int                      `json:"bonding_mode"`
 }
 
-var active_subscription = promauto.NewGauge(prometheus.GaugeOpts{
-	Name: "hegel_active_subscriptions",
+var activeSubscription = promauto.NewGauge(prometheus.GaugeOpts{
+	Name: "hegel_activeSubscriptions",
```

Comment:
This doesn't seem like a necessary change

Reply:
Yes, but I did this because of naming convention in golang.

Reply:
Reverted it.